README.md

```diff
@@ -1,3 +1,23 @@
 # Running Order
-Running Order is an app that allows you to describe the features and the steps to do during a demo for a client.
-It organizes all features to show by categories, and help you to deliver an awesome demo by showing you the steps to do in the app during the demo. 
+A SwiftUI app for helping demonstrate sprint changes, for macOS.
+
+![Image](Images/image1.png?)
+
+## Goal
+When you work in a agile team, you need to show the changes of the finished sprint. This app is designed to help a development team to organize which feature to show, in which order, with some needed informations such as links, usernames, environment names,...
+
+In order to work with multiple members of the team and respect users privacy, all the informations are stored in an iCloud private database owned by a member of the team.
+
+## Roadmap
+Currently this app is at a very early state of development. The first goal is to meet the requirements of one dev team at Worldline, be useful, and prove its value.
+
+The second goal is to be generalized to other iOS development teams, as the app requires a mac to work. Past this point, we'll see if we can launch the app on the mac app store, to reach development teams outside Worldline.
+
+Then, develop a counterpart to reach other interested teams (iOS version, Windows version, or web, in order to work on desired platforms)
+
+## Technical requirements
+Currently the app supports macOS Catalina, but it will be upgraded to Big Sur soon to permit usage of newest SwiftUI changes.
```

Comment:
```suggestion
Currently the app only supports macOS Catalina, but it will be upgraded to Big Sur soon to allow the usage of the newest SwiftUI changes.
```

Reply:
seen together